src/hotspot/share/runtime/safefetch.hpp

```diff
@@ -31,7 +31,7 @@
 // Safefetch allows to load a value from a location that's not known
 // to be valid. If the load causes a fault, the error value is returned.
 
-#ifdef _WIN32
+#if defined(_WIN32) && !defined(_M_ARM64)
   // Windows uses Structured Exception Handling
```

Comment:
```suggestion
#if defined(_WIN32) && !defined(_M_ARM64)
  // Windows x86_64 uses Structured Exception Handling
```
And update the main comment in safefetch_windows.hpp to say the same.